app/assets/stylesheets/components/_dashboard.scss

```diff
@@ -0,0 +1,38 @@
+.dashboard-container {
+  display: flex;
+  background-color: #F1EAF4;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
+  border-radius: 12px;
+  height: 90vh
+}
+
+.col-4 {
+  padding-left: 0px;
+  padding-right: 0px;
+}
+
+.col-8 {
+  padding-right: 0px;
+}
```

Comment:
don't 'overwrite' bootstrap styles like this - you will affect this style everywhere in your app
if someone else ever uses these bootstrap classes in another place in the app it won't work as expected

p.s. the different files for the stylesheets (e.g. dashboard.scss) has no impact on where these styles are applied - styles are always "global". In other words, ANY class you write in ANY file will be loaded on EVERY page

Reply:
Instead, make a new class that is called noPaddingX or use the bootstrap styles for removing horizontal padding `px-0` (I think)